docker/dev/Dockerfile

```diff
@@ -40,13 +40,11 @@ RUN apt-get update \
     libleveldb-dev \
     libmicrohttpd-dev \
     libminiupnpc-dev \
-    libprotobuf-dev \
```

Comment:
Similar packages in `docker/Dockerfile` and `docker/Dockerfile.cuda` should also be removed